onnx/defs/tensor/defs.cc

```diff
@@ -735,7 +735,8 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
-static const char* Scatter_ver9_doc = R"DOC(
+static const char* Scatter_ver11_doc = R"DOC(
+This operator is deprecated and renamed ScatterElements. 
```

Comment:
Minor rewording suggestion: "This operator is deprecated. Please use ScatterElements, which provides the same functionality."